src/scripts/main.js

```diff
@@ -1,8 +1,24 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const arrayPeople = people.map(person => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+  const sex = person.sex === 'm' ? 'Male' : 'Female';
+  const tablePerson = `
+  <tr>
+      <td>${person.name}</td>
+      <td>${sex}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${age}</td>
+      <td>${century}</td>
+   </tr>
+  `;
```

Comment:
```suggestion
  return `
  <tr>
      <td>${person.name}</td>
      <td>${sex}</td>
      <td>${person.born}</td>
      <td>${person.died}</td>
      <td>${age}</td>
      <td>${century}</td>
   </tr>
  `;
```

No need to create the variable just to return it

Reply:
Ok. Thanks.